client/src/pages/searches/Search.tsx

```diff
@@ -885,6 +941,37 @@ const Search = ({
   function closeSaveModal() {
     setShowSaveSearch(false)
   }
+
+  function openExportResultsModal() {
+    setShowExportResults(true)
+  }
+
+  async function exportResultsFromModal() {
+    await exportSearchResults(
```

Comment:
I'd remove the `await` here, so the download can continue in the background and the dialog is closed (and the user can do other things instead of wait for the export to complete).